src/Middleware/SpaServices.Extensions/src/DevelopmentServer/DevelopmentServerMiddleware.cs

```diff
@@ -9,20 +9,24 @@
 using System;
 using System.IO;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.SpaServices.Extensions.Util;
 
-namespace Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer
+namespace Microsoft.AspNetCore.SpaServices.DevelopmentServer
 {
-    internal static class ReactDevelopmentServerMiddleware
+    internal static class DevelopmentServerMiddleware
     {
         private const string LogCategoryName = "Microsoft.AspNetCore.SpaServices";
         private static TimeSpan RegexMatchTimeout = TimeSpan.FromSeconds(5); // This is a development-time only feature, so a very long timeout is fine
 
         public static void Attach(
             ISpaBuilder spaBuilder,
-            string npmScriptName)
+            string npmScriptName,
+            string waitText,
+            Dictionary<string, string> extraArgs,
+            string serverName = "App")
```

Comment:
Does there have to be a default value for this parameter? It would be clearer to make it mandatory if that's possible.